keps/sig-node/3542-cri-image-pulling-with-progress-notification/README.md

```diff
@@ -255,6 +255,19 @@ For kubelet we propose the default granularity to be time-based with 30 seconds
 Pod object will get an event with image pulling progress every 30 seconds.
 The granularity type and interval should be configurable through kubelet-config.
 
+Suggested new Kubelet config fields are these:
+
+      // ImagePullProgressType is the type of ImagePullProgressInterval.
+      // Default: "time"
```

Comment:
I guess we should mention the different types here, time and size (?)

And in the comment of ImagePullProgressInterval we should mention the unit, e.g. seconds if the type is time, and MiB if size

Reply:
I was wrong, there is no values anywhere else, CRI defines enums for serialization, not the values that kubelet-config should parse to select enum values, of course. Added supported values.